src/sphinxql.y

```diff
@@ -654,7 +654,6 @@ filter_expr:
 	
 filter_item:	
 	expr_ident '=' bool_or_integer_value
-
```

Comment:
you should double check the code prior the commit and remove all changes like this space or new line removal.

Reply:
I removed the empty newline because it's more consistent, given the surrounding context - there are no other empty newlines after any top-level item in this file. This is what it looked like before:

```

filter_expr:
	filter_item							{ pParser->SetOp ( $$ ); }
	| filter_expr TOK_AND filter_expr	{ pParser->FilterAnd ( $$, $1, $3 ); }
	| filter_expr TOK_OR filter_expr	{ pParser->FilterOr ( $$, $1, $3 ); }
	| '(' filter_expr ')'				{ pParser->FilterGroup ( $$, $2 ); }
	| knn_item
	;
	
filter_item:	

	expr_ident '=' bool_or_integer_value
		{
			CSphFilterSettings * pFilter = pParser->AddValuesFilter ( $1 );
			if ( !pFilter )
				YYERROR;
			pFilter->m_dValues.Add ( $3.GetValueInt() );
		}
	| expr_ident TOK_NE bool_or_integer_value
		{
			CSphFilterSettings * pFilter = pParser->AddValuesFilter ( $1 );
			if ( !pFilter )
				YYERROR;
			pFilter->m_dValues.Add ( $3.GetValueInt() );
			pFilter->m_bExclude = true;
		}
```

Reply:
(If you would prefer I don't remove this newline, I can revert it btw - I'm just explaining my choice in doing so in this PR)

Reply:
it could be to create minimal difference in the commit related to code and if you think it is better to make other indentation - to create another commit with just indentation changes and comment about that change. 
It could be better not to mix different changes in the one commit

Reply:
Ok, removed the change.